app/src/main/java/com/example/taskyy/data/local/data_access_objects/AgendaActivityDao.kt

```diff
@@ -16,4 +17,16 @@ interface AgendaActivityDao {
     @androidx.room.Query("SELECT * FROM reminder_table WHERE id = :eventID")
     suspend fun getReminderByEventId(eventID: String): ReminderEntity
 
+    @androidx.room.Query("UPDATE reminder_table SET description = :description, time = :time, remindAt = :remindTime, title = :title WHERE id = :eventId")
+    suspend fun updateReminder(
+        description: String,
+        time: Long,
+        title: String,
+        remindTime: Long,
+        eventId: String
+    )
```

Comment:
Two things:
1. Why don't you just expose a single `@Upsert` function which lets you insert and update events in one function? I find passing those individual fields rather complicated, since you could just insert a whole entity with the changed data. If you insert that with an ID that exists in the DB and you use `@Upsert`, it will replace the existing entity
2. I think you mean `reminderId` here for the parameter